.github/CONTRIBUTING.md

```diff
@@ -103,68 +101,66 @@ Additional references:
-
+  See [Contributing to documentation related to PowerShell](#contributing-to-documentation-related-to-powershell).
 * Add a meaningful title of the PR describing what change you want to check in.
   Don't simply put: "Fixes issue #5".
   A better example is: "Add Ensure parameter to New-Item cmdlet", with "Fixes #5" in the PR's body.
-
 * When you create a pull request,
   including a summary of what's included in your changes and
   if the changes are related to an existing GitHub issue,
   please reference the issue in pull request description (e.g. ```Closes #11```).
   See [this][closing-via-message] for more details.
-
 * If the change warrants a note in the [changelog](../CHANGELOG.MD)
   either update the changelog in your pull request or
   add a comment in the PR description saying that the change may warrant a note in the changelog.
   New changes always go into the **Unreleased** section.
   Keeping the changelog up-to-date simplifies the release process for Maintainers.
-  An example:
-    ```
-    Unreleased
-    ----------
-
-    * `Update-Item` now supports `-FriendlyName`.
-    ```
-    Please use the present tense and imperative mood when describing your changes:
-
-      * Instead of "Adding support for Windows Server 2012 R2", write "Add support for Windows Server 2012 R2".
-
-      * Instead of "Fixed for server connection issue", write "Fix server connection issue".
-
-    This form is akin to giving commands to the code base,
-    and is recommended by the Git SCM developers.
-    It is also used in the [Git commit messages](#common-engineering-practices).
-
-    Also, if change is related to a specific resource, please prefix the description with the resource name:
-
-      * Instead of "New,parameter 'ConnectionCredential' in New-SqlConnection",
-        write "New-SqlConnection: added parameter 'ConnectionCredential'".
+  An example (with an associated PR #):
+
+  ```markdown
+  Unreleased
+  ----------
+
+  * `Update-Item` now supports `-FriendlyName` (#1234).
+  ```
+
+* Please use the present tense and imperative mood when describing your changes:
+  * Instead of "Adding support for Windows Server 2012 R2", write "Add support for Windows Server 2012 R2".
+  * Instead of "Fixed for server connection issue", write "Fix server connection issue".
+
+  This form is akin to giving commands to the code base
+  and is recommended by the Git SCM developers.
+  It is also used in the [Git commit messages](#common-engineering-practices).
+* If the change is related to a specific resource, please prefix the description with the resource name:
+  * Instead of "New parameter 'ConnectionCredential' in New-SqlConnection",
+  write "New-SqlConnection: added parameter 'ConnectionCredential'".
+* If your change warrants an update to user-facing documentation,
+  a Maintainer will add the `Documentation Needed` label to your PR and add an issue to the [PowerShell-Docs repository][PowerShell-Docs],
```

Comment:
Unfortunately, there's not a stable set of folks or workflow for getting these docs written (and actually, a lot of the preliminary writing will fall on engineers/me on the PowerShell Team), so we have to do this asynchronously with potentially extended latency. 

We'll also have a lot of these things that are more than just a parameter addition (e.g. the `&` backgrounding) where we'll need some longer-form content. I just don't want these tasks to get lost in merged PRs.